src/util.cc

```diff
@@ -423,21 +427,82 @@ const char* SpellcheckString(const char* text, ...) {
 }
 
 #ifdef _WIN32
+std::wstring Utf8ToWide(const std::string& u8_string) {
+  if (u8_string.empty())
+    return std::wstring();
+
+  size_t size_needed =
+      MultiByteToWideChar(CP_UTF8, 0, u8_string.data(),
+                          static_cast<int>(u8_string.size()), NULL, 0);
```

Comment:
assert that `u8_string.size()` is within the range supported by `int`

Reply:
Do I need to make this check? The size of the string must be ~2 GB (32-bit) to overflow the integer. Isn't this primarily for dealing with paths, which on windows is limited to 255 chars max, or am I missing something here?

Reply:
IMHO it's fine.

Reply:
I strongly disagree. An assert from #<cassert> will only be compiled in during a debug build, but will provide a defense against pathologically broken inputs, at the cost of one line of code.

Better safe than sorry.

Reply:
@jlonnberg please ensure you use an assert where you convert integral types.